.github/workflows/e2e_tests.yml

```diff
@@ -4,13 +4,16 @@ on:
   deployment_status:
 
 jobs:
-  e2e-tests:
+  _e2e-tests:
```

Comment:
With this approach, we now end up with four separate status checks per test run, compared to two in the previous setup.
While this may seem a bit redundant, we believe it's a worthwhile tradeoff to avoid false positives in the PR checks.


<img width="391" alt="1" src="https://github.com/user-attachments/assets/32526875-2587-44bc-afbc-d12059989eb6" />

<img width="395" alt="2" src="https://github.com/user-attachments/assets/6514b02b-221b-4631-afb0-c9aae2d0d7f1" />